ui.apps/src/main/content/jcr_root/apps/core/fd/components/formsportal/link/v2/link/link.html

```diff
@@ -0,0 +1,32 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div class="cmp-link">
```

Comment:
How to reuse the markup from v1 ? Why have you copy pasted v1 here ? Ideally, markup is still the same. this also holds true for other copy pasted files from v1

Reply:
Agreed. I'll update v2 to refer re-usable components from v1.
I was looking at list v1 and list v2 which is a copy pase, so I copied these here.